### PR TITLE
Simplify define calls

### PR DESCRIPTION
**Pull request by github-nilsson**
Assuming Object.defineProperty exists and works, as is the case in all target browsers, we don't need a fallback for extending String for fromCodePoint and codePointAt.

### CODE REVIEW

File: string.polyfill.js [hunk 1]
```diff
@@ -17,18 +17,12 @@
...
-        var result = $defineProperty(object, object, object) && $defineProperty;
-      } catch (error) {}
-      return result;
-    })();
-    var stringFromCharCode = String.fromCharCode;
-    var floor = Math.floor;
-    var fromCodePoint = function () {
+  var stringFromCharCode = String.fromCharCode;
+  var floor = Math.floor;
+  Object.defineProperty(String.prototype, 'fromCodePoint', {
```
Comment (behnammodi) [inline string.polyfill.js:22]: `fromCodePoint` is not a prototype

Comment (github-nilsson) [reply string.polyfill.js]: Too quick copy and paste.

Comment (behnammodi) [reply string.polyfill.js]: Do you think I need to check the previous PR :) ?

**Comment by behnammodi** [comment]
@github-nilsson Could you please resolve the conflict?

**Pull request merged**